java/ql/src/experimental/Security/CWE/CWE-555/PasswordInConfigurationFile.qhelp

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+      Storing a plaintext password in a configuration file allows anyone who can read the file to access the password-protected resources. Therefore it is a common attack vector.
```

Comment:
```suggestion
      Storing a plaintext password in a configuration file allows anyone who can read the file to access the password-protected resources.
```

Reply:
Thanks @smowton for reviewing this PR. I've made the text change.